architecture-baseline.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Did u run this from suite nonsplit?

Reply:
No, this file is just needed for Spryker CI.

Reply:
Sure, it is needed for CI, but it should be generated either as valid one or not at all.
This creates the false sense of "all green".
Best to quickly run this from nonsplit by adding the module tmp.

Reply:
`console code:sniff:architectur -u -m SprykerEco.AvalaraTax` creates the same.